### PR TITLE
Layout component in React

### PR DESCRIPTION
**Pull request by matzimowski**
- `<Layout>, <LayoutContent>, <LayoutAsideContent>, <LayoutBox>`
- no visual changes

### CODE REVIEW

File: src/components/layout/pages/layout_large.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import LayoutAsideContent from './../LayoutAsideContent';
+import LayoutFooter from './../LayoutFooter';
+import Text from '../../text/Text';
+
+const LayoutLarge = () =>
+  <html>
+  <head>
+    <meta charSet="utf-8"/>
+    <link rel="stylesheet" href="../../../style-guide.css"/>
+    <style>{"\
```
Comment (matikrk) [inline src/components/layout/pages/layout_large.jsx]: use string template

File: src/components/layout/pages/layout_medium.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import LayoutAsideContent from './../LayoutAsideContent';
+import LayoutFooter from './../LayoutFooter';
+import Text from '../../text/Text';
+
+const LayoutMedium = () =>
+  <html>
+  <head>
+    <meta charSet="utf-8"/>
+    <link rel="stylesheet" href="../../../style-guide.css"/>
+    <style>{"\
```
Comment (matikrk) [inline src/components/layout/pages/layout_medium.jsx]: same here, string template

File: src/components/layout/pages/layout_large.jsx (hunk 1 above)
Comment (matzimowski) [inline src/components/layout/pages/layout_large.jsx]: i put in in one line now

File: src/components/layout/pages/layout_medium.jsx (hunk 2 above)
Comment (matzimowski) [inline src/components/layout/pages/layout_medium.jsx]: i put in in one line now

**Comment by matikrk** [comment]
Imho you can merge 3 components into 1
Layout, LayoutContainer, LayoutFooter
and then children of Layout will be children of LayoutContainer 
new created props footer will be children of LayoutFooter

Additionaly i would create props header to pass headerContent.

File: src/components/layout/pages/layout_small.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <body>
+      <Layout>
+        <div className="sg-header sg-header--fixed">
+          <div className="sg-header__container">
+            Header
+          </div>
+        </div>
+        <LayoutContainer>
+          <LayoutContent>
+            <LayoutBox>
```
Comment (kdzwinel) [inline src/components/layout/pages/layout_small.jsx]: is layoutbox always inside layoutcontent? If so, shouldn't we merge them?

Comment (matzimowski) [inline src/components/layout/pages/layout_small.jsx]: not sure, in examples, we have layout-box in layoutAsideContent

**Pull request merged**